lib/matplotlib/tests/test_tightlayout.py

```diff
@@ -157,6 +158,15 @@ def test_tight_layout8():
     example_plot(ax, fontsize=24)
 
 
+@image_comparison(baseline_images=['tight_layout9'])
+def test_tight_layout9():
```

Comment:
There are now two test functions in one file with the same name. So the first one gets clobbered by the second one.